### ISSUE TITLE
Why openai opensource Grok? not XAI?

### ISSUE THREAD

**Issue by K-tang-mkv** [openai/grok#10]

**Comment by Harsh2509** [comment]
I am also curious to know what is happening..

**Comment by MING-ZCH** [comment]
lol.

**Comment by scris** [comment]
You're humorous.

**Comment by JamesFlare1212** [comment]
Indeed, the Grok repository has been in existence for many years, originating from the OpenAI paper "Grokking: Generalization Beyond Overfitting on Small Algorithmic Datasets" in 2022. Although it's not the same Grok as the large language model Grok of xAI, it did indeed precede Musk's open-source Grok.
